src/machine/machinectl.c

```diff
@@ -386,6 +389,9 @@ static int list_images(int argc, char *argv[], void *userdata) {
         if (!table)
                 return log_oom();
 
+        if (arg_full)
+                table_set_width(table, 0);
```

Comment:
I get the following:
```console
$ build/machinectl --full
MACHINE CLASS     SERVICE        OS     VERSION ADDRESSES       
rawhide container systemd-nspawn fedora 32      192.168.122.249…

1 machines listed.
```
... i.e. there's still ellipsization.

Reply:
that ellipsation is not done by the table logic, but by machinectl itself, it only shows one ip address by default, see the --max-addresses=

One could argue though that --full should influence --max-addresses= too, but that#s probably a separate commit

Reply:
also, the table formatting code should probably wisen up and support multi-line fields sanely, to cover for the IP address case better...